src/adapter/src/session.rs

```diff
@@ -1063,15 +1063,16 @@ impl<T: TimestampManipulation> TransactionStatus<T> {
         }
     }
 
-    /// Adds operations to the current transaction. An error is produced if
-    /// they cannot be merged (i.e., a timestamp-dependent read cannot be
-    /// merged to an insert).
+    /// Adds operations to the current transaction. An error is produced if they cannot be merged
+    /// (i.e., a timestamp-dependent read cannot be merged to an insert).
+    ///
+    /// The `DDL` variant is an exception and does not merge operations, but instead overwrites the
+    /// old ops with the new ops.
```

Comment:
Looks like this is existing behavior, but it'd probably be useful to explain why this is correct.